app/pages/patientdata/patientdata.js

```diff
@@ -76,7 +76,7 @@ var PatientData = React.createClass({
         },
         timePrefs: this.props.timePrefs
       },
-      chartType: 'daily',
+      chartType: 'modal',
```

Comment:
ha ha - ok just viewed the whole file and spotted its set in the getInitialState